spec/helpers/project_helper.rb

```diff
@@ -55,6 +55,16 @@ def create_random_user(client, opts = {})
           }
           GoodData::Membership.create(opts, client: client)
         end
+
+        def load_full_project_implementation(client)
+          spec = JSON.parse(File.read("./spec/data/blueprints/test_project_model_spec.json"), :symbolize_names => true)
+          domain = client.domain(ConnectionHelper::DEFAULT_DOMAIN)
+          blueprint = GoodData::Model::ProjectBlueprint.new(spec)
+
+          project = client.create_project_from_blueprint(blueprint,
+            token: ConnectionHelper::GD_PROJECT_TOKEN, environment: ProjectHelper::ENVIRONMENT)
```

Comment:
Style/AlignParameters: Align the parameters of a method call if they span more than one line.